core-model/src/main/java/com/sequenceiq/cloudbreak/domain/organization/OrganizationAwareResource.java

```diff
@@ -12,4 +12,6 @@
     void setOrganization(Organization organization);
 
     OrganizationResource getResource();
+
+    String getOwner();
```

Comment:
The old conception and new conception are mixed. We should separate this function into another interface

Reply:
Later on we are going to eliminate owner totally as i know. On the other hand `OrganizationAwareResource` is a key interface in the system, and i'm afraid to change in the last minute.

Reply:
Big question is how are we planning to eliminate owner if we are commited to support cloudbreak upgradability?